yt_dlp/extractor/tenplay.py

```diff
@@ -103,13 +103,13 @@ def _real_extract(self, url):
             f'https://vod.ten.com.au/api/videos/bcquery?command=find_videos_by_id&video_id={data["altId"]}',
             content_id, 'Downloading video JSON')
         m3u8_url = self._request_webpage(
-            HEADRequest(video_data['items'][0]['HLSURL']),
+            HEADRequest(video_data['items'][0]['dashManifestUrl'].replace('manifest=mpeg-dash', 'manifest=m3u')),
             content_id, 'Checking stream URL').url
         if '10play-not-in-oz' in m3u8_url:
             self.raise_geo_restricted(countries=['AU'])
```

Comment:
```suggestion
            self.raise_geo_restricted(countries=['AU'])
         if '10play_unsupported' in m3u8_url:
             raise ExtractorError("Looks like you're using an older version of the 10 play app, which is now unsupported")

```

If this happens again, this should hopefully prevent yt-dlp downloading useless 10 second files

Reply:
Maybe add ` (10Play says "unsupported")` to the error message?

Reply:
Sounds reasonable, I do like @dirkf 's suggestion for a more specific error message than a generic unable to extract stream error.